### DISCUSSION TITLE
Symlinks are broken in Vercel integration for Github?

### DISCUSSION THREAD

**Discussion by artecoop** [vercel/vercel#5347] [Help] [answered]
Imagine having this github repo
---/
------> react_app_folder
------> common_folder
------> next_app_folder

Everything I need to share among projects are inside the Common folder. I symlink `common_folder` inside my other projects like:
```
cd react_app_folder
ln -s ../common_folder .
```
I can develop without problems or warning from eslint or any other tool (in fact my projects are in typescript for example).
Now I push all on my github repo. Standard stuff. Here Github recognize the symlink and show me the correct path if opened.

Time to upload to Vercel. In dashboard I hit "import project" and input the url of my repo. Here `common_folder` is recognized by the file manager as file, and deploy fails because the build stack does not follow the symlink, thus losing reference to the common folder.

A month ago, in another project with the very same logic, all worked like a charm.

Is something changed or I'm doing something wrong and I was very lucky?

**Comment by maiertech** [comment]
I just noticed the same problem in a project where I symlink a bunch of content MDX files into different example sites. `dev` and `build` work fine locally, but the build breaks on Vercel. I can see in the log that the symlink is not honored.

**Comment by mikestopcontinues** [comment, marked as answer]
I just received word from Vercel support. Symlinks are simply not supported during build.
